README.md

```diff
@@ -25,146 +25,165 @@ Then you can run docker image with:
+  $ python manage.py responsiblepersons --file path/to/filename.xlsx
   ```
+  
 
-Projects are also synced to PW service when a PATCH request is made to the projecs endpoint.
+---
 
-### Excel Files
 
-Project data and finances can be imported using excel files into the infra tool.
+## Managing project packages
 
-Import Location/Class hierarchy structure:
+- We use `pip` to manage python packages we need
+- After adding a new package to requirements.txt file, compile it and re-build the Docker image so that the container would have access to the new package
 
   ```bash
-  $ python manage.py hierarchies --file path/to/hierarchy.xlsx
+  docker-compose up --build
   ```
 
-Import Planning (TS) project data:
+## Running tests
+
+Tests are written for django management commands and the endpoints. They can be found in the following location:
 
   ```bash
-  $ python manage.py  projectimporter --import-from-plan path/to/planningFile.xlsx
+  infraohjelmointi_api/tests
   ```
-
-Import Budget (TAE) project data:
+Run the tests
 
   ```bash
-  $ python manage.py  projectimporter --import-from-budget path/to/budgetFile.xlsx
+  $ python manage.py test
   ```
-
-Import Planning (TS) and Budget (TAE) files in bulk together:
+An optional verbosity parameter can be added to get a more descriptive view of the tests
 
   ```bash
-  $ ./import-excels.sh -d path/to/directory/containing/all/Excels
+  $ python manage.py test -v 1/2/3
   ```
 
-# Fix Database Issues
+## How to: production release
 
-Imported files might included typos or incorrectly written information.
+1. Create a release PR from develop to main
+2. Wait for the PR pipeline to run and check that all checks pass
+3. Merge the PR
+4. Trigger build-infraohjelmointi-api-stageprod
+5. Approve pipeline run in azure. Deploy pipelines are triggered by the build pipeline but prod deploy needs to be approved separately (=2 approvals in total). To approve, open the pipeline run you want to approve (from menu, select pipelines, then select the correct pipeline and then select the run you need to approve) and there should be a button to approve it (pipeline run is paused until you approve).
 
-Fix known issues from database:
+## External data sources
+
+Infra tool project data and financial data can be imported from external sources.
+
+### SAP
+
+To synchronize project data with SAP in local environment, VPN service provided by platta should be running.
+
+Populate DB with SAP costs and commitments using management command:
 
   ```bash
-  $ psql $DATABASE_URL
-  infraohjelmointi_api_db=# \i fix-database.sql
+  $ python manage.py sapsynchronizer
   ```
-
-Update projects' missing `projectDistrict_id` data with `infraohjelmointi_api_projectdistrict.id`.
+All projects in DB will also be synced with SAP to update SAP costs and commitments at midnight through the script:
 
   ```bash
-  $ psql $DATABASE_URL
-  infraohjelmointi_api_db=# \i update-districts.sql
+  $ ./sync-from-sap.sh
```

Comment:
I guess that this is used only in the production environment? We could mention that there and also some instructions about the usage of this e.g. if we use this only when the client asks for it, to fix some specific problem or if there are some cases in the production release that require this to the run etc

Reply:
The file will be run on prod and dev environment with CRON job, and it will be run everyday at midnight. At the moment it does not work locally with VPN.

I will add a mention about cron jobs and how these are run every night on both prod and dev environments. It's good idea to add cases here too, but I'm not sure what kind of example cases we could add here. 🤔